preview/docs-ref-mapping/reference-preview.yml

```diff
@@ -70,6 +70,26 @@
       href: ~/docs-ref-services/core-experimental-readme-pre.md
       children:
       - com.azure.core.experimental*
+    - name: Gson Serializer 
+      uid: azure.java.sdk.landingpage.services.sdk.core.GsonSerializer
```

Comment:
Updating preview manually can be finicky. Given that UIDs here don't have `.pre` on them, I'm betting you did this manually yes? That's totally ok, but there are two possible easier solutions.

1. Only update `reference-unified`. Next CI run will update `reference-preview` for you.
2. Update `reference-unified`, then locally invoke the script that CI uses to update the the ToCs. That'll also modify it for you!

I can walk you through either one! Gotchas, etc